sapp/ui/filter_predicates.py

```diff
@@ -85,7 +85,8 @@ def __init__(self, column: Union[Column[_T], DBID], items: Sequence[_T]) -> None
 
     def apply(self, query: Query[_Q]) -> Query[_Q]:
         # pyre-ignore: SQLAlchemy too dynamic.
-        return query.filter(or_(*[self._column.like(item) for item in self._items]))
```

Comment:
What's the reason for this deletion? It seems to be exactly the same as the code block two lines below it:
```
x = or_(*[self._column.like(item) for item in self._items])
return query.filter(x)
```

Reply:
None, sorry this file wasn't to be added to the commit